beam/hooks.py

```diff
@@ -625,7 +625,7 @@
 			"path": "/repack",
 			"name": "repack",
 			"component": "Repack",
-			"meta": {"requiresAuth": True, "doctype": "Stock Entry", "view": "list"},
+			"meta": {"requiresAuth": True, "doctype": "Stock Entry", "view": "form"},
```

Comment:
@lauty95 I had to change the setup here, since trying to scan an item was somehow leading to the `filter` action. Not sure how it was working for you locally though :thinking: